controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java

```diff
@@ -477,6 +565,19 @@ public static PersistentResourceXMLBuilder builder(final PathElement pathElement
         return new PersistentResourceXMLBuilder(pathElement, namespaceURI);
     }
 
+    /**
+     * Creates builder for passed path element
+     *
+     * @param elementName name of xml element that is used as decorator
+     * @return PersistentResourceXMLBuilder
+     * @deprecated Not decided yet if this should become official API!
```

Comment:
We can't have stuff like this if we are going to be rapidly releasing. It's also not used in the PR. Please remove this method.

Reply:
This one is actually used quite heavily in this PR. 
But looking forward I would love to remove need for it by refactoring xml model of elytron subsystem.
I've updated deprecated comment for time being.
The other deprecated method #addChild(final String xmlElementName, ResourceParser parser, ResourceMarshaller marshaller)
I've commented out for now.